docs/core/install/linux-package-mixup.md

```diff
@@ -1,14 +1,14 @@
 ---
-title: Troubleshoot .NET Package Mix ups on Linux
-description: Learn about how to troubleshoot strange .NET package errors on Linux.
+title: Troubleshoot .NET package mix ups on Linux
+description: Learn about how to troubleshoot strange .NET package errors on Linux. These errors may occur when you run the dotnet command.
 author: omajid
 ms.date: 10/31/2022
 no-loc: ['usr','lib64','share','dotnet','libhostfxr.so', 'fxr', 'FrameworkList.xml', 'System.IO.FileNotFoundException']
 ---
 
-# Troubleshoot _fxr_, _libhostfxr.so_, and _FrameworkList.xml_ errors
+# Troubleshoot .NET errors related to missing files on Linux
```

Comment:
@mirespace @richlander Do you folks want to add the steps from https://github.com/dotnet/core/issues/7699 to this doc?

Reply:
Nevermind, I am an idiot.